src/Color.purs

```diff
@@ -160,22 +163,40 @@ rgba' r g b a = rgba (round $ r * 255.0)
 rgb' :: Number -> Number -> Number -> Color
 rgb' r g b = rgba' r g b 1.0
 
--- | Create a `Color` from hue, saturation, lightness and alpha values. The
+-- | Create a `Color` from Hue, Saturation, Lightness and Alpha values. The
```

Comment:
Because `and Value values` looks better then `and value values`. will fix other occurrences.